Sources/MeiliSearch/Documents.swift

```diff
@@ -14,22 +14,22 @@ struct Documents {
     self.request = request
   }
 
-  // MARK: Query
+  // MARK: Path
 
   func get<T>(
     _ uid: String,
     _ identifier: String,
     fields: [String]? = nil,
     _ completion: @escaping (Result<T, Swift.Error>) -> Void)
   where T: Codable, T: Equatable {
-    var query: String = "/indexes/\(uid)/documents/\(identifier)"
+    var path: String = "/indexes/\(uid)/documents/\(identifier)"
 
     if fields != nil {
-      let fieldsQuery = "?fields=\(fields?.joined(separator: ",") ?? "")"
-      query.append(fieldsQuery)
+      let fieldsPath = "?fields=\(fields?.joined(separator: ",") ?? "")"
+      path.append(fieldsPath)
```

Comment:
In this case the "query" name is correct :)
```suggestion
      let fieldsQuery = "?fields=\(fields?.joined(separator: ",") ?? "")"
      path.append(fieldsQuery)
```